### PR TITLE
Pp 9033/update docs

### PR DESCRIPTION
**Pull request by jkadluczka**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

**Comment by ale-kacperczyk** [comment]
I think we don't need the commit from PP-8966 in this PR as it was already merged

**Comment by jkadluczka** [comment]
I mean I can cherrypick this but there is no change in this pr besides relevant to the issue.

File: src/pages/monetization/partner-program-api/index.mdx [hunk 1]
```diff
@@ -800,6 +800,7 @@ curl --request DELETE \
 - [Get Earnings Metrics](#get-earnings-metrics)
 - [Get Conversions Metrics](#get-conversions-metrics)
 - [Get Clicks Metrics](#get-clicks-metrics)
+- [Get Referred Partners Commission Mertics](#get-referred-partners-commission-metrics)
```
Comment (fabcos92) [inline src/pages/monetization/partner-program-api/index.mdx:803]:
```suggestion
- [Get Referred Partners Commission Metrics](#get-referred-partners-commission-metrics)
```

File: src/pages/monetization/partner-program-api/index.mdx [hunk 2]
```diff
@@ -1533,6 +1534,61 @@ curl --request POST \
 
 <Text>
 
+## Get Referred Partners Commission Mertics
```
Comment (fabcos92) [inline src/pages/monetization/partner-program-api/index.mdx:1537]:
```suggestion
## Get Referred Partners Commission Metrics
```

**Review by fabcos92 — APPROVED** [review]

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**